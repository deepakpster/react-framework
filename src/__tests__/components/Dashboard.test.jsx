import React from 'react';
import renderer from 'react-test-renderer';
import Dashboard from './../../js/components/dashboard';

test('Link changes the class when hovered', () => {
	const component = renderer.create(<Dashboard title="Dashboard (TEST)" />);
	const tree = component.toJSON();
	expect(tree).toMatchSnapshot();
	// manually trigger the callback
	// tree.props.onMouseEnter();
	// re-rendering
	// tree = component.toJSON();
	// expect(tree).toMatchSnapshot();

	// manually trigger the callback
	// tree.props.onMouseLeave();
	// re-rendering
	// tree = component.toJSON();
	// expect(tree).toMatchSnapshot();
});
