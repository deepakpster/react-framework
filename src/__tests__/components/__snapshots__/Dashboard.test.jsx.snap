// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<div>
  <h1>
    Dashboard (TEST)
  </h1>
  <hr />
  <p>
    lorem posum item blah blah blah
  </p>
</div>
`;
